addons/mail/tests/test_mail_channel.py

```diff
@@ -315,6 +340,17 @@ def test_channel_info_get(self):
         same_solo_channel_info = self.env['mail.channel'].channel_get(partners_to=self.partner_employee_nomail.ids)
         self.assertEqual(same_solo_channel_info['id'], solo_channel_info['id'])
 
+    @users('employee')
+    def test_channel_info_get_should_update_last_activity_time(self):
```

Comment:
In general this makes no sense for a "get" to update anything. This comes from its "pinned" part, which should be mentioned in the name. (note: the "get" name was originally bad choice there)

Reply:
I added a comment above the test. It is a bit difficult to explain in the function name...